.github/workflows/npm.yml

```diff
@@ -7,7 +7,14 @@ on:
     branches: [master]
 
 jobs:
-  build:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - run: npm ci
+      - run: npm run lint
```

Comment:
this is already run as part of `npm test`: https://github.com/mixpanel/mixpanel-js/blob/475b2c6853dd49ffc625daac3925bf37940ec390/package.json#L21

Reply:
Ah, I missed that. Do you want to remove `npm run lint` from the `test` script? I think it's a bit unexpected.

Feel free to close this pull request if there are no changes here that you like.